src/test/java/racingcar/controller/GameControllerRestAssuredTest.java

```diff
@@ -0,0 +1,60 @@
+package racingcar.controller;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import racingcar.TestDatabaseConfig;
+import racingcar.controller.dto.PlayRequest;
+
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.is;
+
+@TestDatabaseConfig
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class GameControllerRestAssuredTest {
+
+    @LocalServerPort
+    int port;
+
+    @BeforeEach
+    void setup() {
+        RestAssured.port = port;
+    }
+
+    @Test
+    @DisplayName("post(/plays) 테스트")
+    void post_plays() {
+
+        RestAssured.given().log().all()
+                .accept(MediaType.APPLICATION_JSON_VALUE)
+                .contentType(ContentType.JSON)
+                .body(new PlayRequest("aa", 1))
+                .when().post("/plays")
+                .then().log().all()
+                .statusCode(HttpStatus.OK.value())
+                .body("winners", hasItems("aa"))
+                .body("racingCars.size()", is(1));
+
+    }
+
+    @Test
+    @DisplayName("post(/plays) GameException bad request 테스트")
+    void post_plays_GameException_bad_request() {
+        RestAssured
+                .given().log().all()
+                .contentType(ContentType.JSON)
+                .body(new PlayRequest("aa,aa", 1))
+
+                .when()
+                .post("/plays")
+
+                .then().log().all()
```

Comment:
```suggestion
                .body(new PlayRequest("aa,aa", 1))
                .when()
                .post("/plays")
                .then().log().all()
```

중간에 개행이 하나씩 들어갔네요 ㅎㅎ

Reply:
반영했습니다!